index.bs

```diff
@@ -197,12 +197,14 @@ The {{XRFrame/createAnchor(pose, space)}} method, when invoked on an {{XRFrame}}
     1. Add [=update anchors=] algorithm to |session|’s [=XRSession/list of frame updates=] if it is not already present there.
     1. Let |device| be |session|'s [=XRSession/XR device=].
     1. Let |effective origin| be |space|'s [=XRSpace/effective origin=].
-    1. Let |anchor native origin| be a new native origin returned from the |device|'s call to create a new anchor using |pose|, interpreted as if expressed relative to |effective origin|.
+    1. Let |anchor native origin| be a new native origin returned from the |device|'s call to create a new anchor using |pose|, interpreted as if expressed relative to |effective origin| at the |frame|'s [=XRFrame/time=].
     1. [=Create new anchor object=] |anchor| using |anchor native origin| and |session|.
     1. Add |anchor| to |session|'s [=XRSession/set of tracked anchors=].
     1. Add a mapping from |anchor| to |promise| to |session|'s [=XRSession/map of new anchors=].
 </div>
 
+Note: It is the responsibilty of user agents to ensure that the initial anchor position aligns as closely as possible with the pose passed in to the {{XRFrame/createAnchor(pose, space)}} call at the time represented by the frame on which the method is called. Specifically, this means that for spaces that are dynamically changing, user agents should attempt to capture the state of such spaces at the said time. This text is non-normative, but expresses the intent of the specification author(s) and contributors and thus it is highly recommended that it is followed by the implementations to ensure consistent behavior across different vendors.
```

Comment:
nit: wondering if this makes it more clearer. Instead of "initial anchor position",  can we use "tracked/created anchor pose"

Reply:
I would avoid saying "tracked anchor pose" or such, as it implies that the "pose" you pass in relative to the reference space is what is durable over time, when it's the opposite - the app is requesting that this location in the world is tracking independently from all other spaces, with the pose specified in the call just used for initially seeding that anchor.

Perhaps:
```suggestion
Note: It is the responsibility of user agents to ensure that the physical origin tracked by the anchor returned by each {{XRFrame/createAnchor(pose, space)}} call aligns as closely as possible with the physical location of `pose` within `space` at the time represented by the frame on which the method is called. Specifically, this means that for spaces that are dynamically changing, user agents should attempt to capture the native origin of such spaces at the app's specified time. This text is non-normative, but expresses the intent of the specification author(s) and contributors and thus it is highly recommended that it is followed by the implementations to ensure consistent behavior across different vendors.
```

Reply:
Thanks for the feedback! @thetuvix - your suggestion looks good so I'm going to accept it, we can always wordsmith it further later, especially since this is a non-normative text so as long as the intent is clear to everyone involved, we should be good here.